### PR TITLE
[14.0] Add pdf_helper: stop copy/pasting code :)

### PR DESCRIPTION
**Pull request by simahawk**
While working on https://github.com/OCA/edi/pull/595 I've found that in sale_order_import the PDF support is broken and going down the rabbit hole I found we have duplicated pieces of code to handle PDFs which are not even tested.

This module aims to provide a tool that can be shared across different modules.
In other repos we have some copy/pasted methods for splitting PDF pages: we might want to move the here too.

### CODE REVIEW

File: pdf_helper/utils.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        """
+        res = {}
+        with BytesIO(self.pdf_file) as fd:
+            xmlfiles = self._extract_xml_files(fd)
+            for filename, xml_obj in xmlfiles.items():
+                root = self._extract_xml_root(xml_obj)
+                if not root:
+                    continue
+                res[filename] = root
+                _logger.debug(
```
Comment (simahawk) [inline pdf_helper/utils.py]: this logging must be removed

**Review by TDu — APPROVED** [review]

**Review by SilvioC2C — APPROVED** [review]
LGTM

**Review by ivantodorovich — APPROVED** [review]
LG

**Pull request merged**